src/index.ts

```diff
@@ -280,7 +280,7 @@ export const syncLogger = (logger: Logger, namespace: string, canForceWrite?: bo
 
         logger.isLevelEnabled = (level) => enabledLevels[level]
     }
-
+    logger.canForceWrite = canForceWrite
```

Comment:
I think we need to check `canForceWrite` before do this?

```js
if(canForceWrite) logger.canForceWrite = canForceWrite
```
Because if canForceWrite is not true, we need to do nothing.

Reply:
not true = undefined = false so it will be correct

Reply:
Yes, I known that. But we have already a falsy value of `canForceWrite` (because the delete method). So why we need to new set `canForceWrite` if it is always `falsy`?

In the code of Florian, the force logging change only when `forceLogging equals true`.

Reply:
I juste think that `if` is better performance than `assign value to object`. But it is very tiny. I close my thread.

Reply:
i think if we can avoid a `if` it's better